doc/api/buffer.markdown

```diff
@@ -392,6 +392,19 @@ byte from the original Buffer.
     // abc
     // !bc
 
+
+### buf.indexOf(value[, byteOffset])
+
+* `value` String, Buffer or Number
+* `byteOffset` Number, Optional, Default: 0
+* Return: Number
+
+Operates similar to
+[Array#indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf).
```

Comment:
Unfortunately that removes the code formatting to let the user know that the text is a link. Technically the correct output would have been
```
`[Array#indexOf()](...)`
```
to place the `<code></code>` around the `<a></a>`, instead of the other way around, but markdown wouldn't be interpreted that way.